src/GongSolutions.WPF.DragDrop/DefaultDropHandler.cs

```diff
@@ -254,20 +254,20 @@ protected static bool IsChildOf(UIElement targetItem, UIElement sourceItem)
 
         protected static bool TestCompatibleTypes(IEnumerable target, object data)
         {
-            TypeFilter filter = (t, o) => { return (t.IsGenericType && t.GetGenericTypeDefinition() == typeof(IEnumerable<>)); };
+            bool InterfaceFilter(Type t, object o) => (t.IsGenericType && t.GetGenericTypeDefinition() == typeof(IEnumerable<>));
 
-            var enumerableInterfaces = target.GetType().FindInterfaces(filter, null);
+            var enumerableInterfaces = target.GetType().FindInterfaces(InterfaceFilter, null);
             var enumerableTypes = from i in enumerableInterfaces
                                   select i.GetGenericArguments().Single();
 
-            if (enumerableTypes.Count() > 0)
+            if (enumerableTypes.Any())
             {
                 var dataType = TypeUtilities.GetCommonBaseClass(ExtractData(data));
                 return enumerableTypes.Any(t => t.IsAssignableFrom(dataType));
             }
             else
             {
-                return target is IList;
+                return target is IList || target is IEditableCollectionView;
```

Comment:
@dauthleikr Thx for your contribution! I think it would be better to use the `ICollectionView` interface here. Because it is used later to get the list from. If we use onle the `IEditableCollectionView` then maybe we don't catch all possible views.

https://github.com/punker76/gong-wpf-dragdrop/blob/d8545166eb08e4d71fc2d2aa67713ba7da70f92c/src/GongSolutions.WPF.DragDrop/Utilities/TypeUtilities.cs#L68

https://github.com/punker76/gong-wpf-dragdrop/blob/d8545166eb08e4d71fc2d2aa67713ba7da70f92c/src/GongSolutions.WPF.DragDrop/DefaultDropHandler.cs#L183

Reply:
Sounds good, I changed it